libraries/animation/src/AnimInverseKinematics.cpp

```diff
@@ -158,6 +159,16 @@ void AnimInverseKinematics::setTargetVars(const QString& jointName, const QStrin
     }
 }
 
+bool debounceJointWarnings() {
+    static QTime last_call;
+
+    if (last_call.elapsed() >= WARNING_DEBOUNCE_TIME || !last_call.isValid()) {
```

Comment:
Should need to test `!last_call.isValid()` if you include the following code in the `AnimInverseKinematics::AnimInverseKinematics()` constructor:
```
debounceJointWarningsClock.start();
```